rust/plugin-lib/src/view.rs

```diff
@@ -131,6 +131,15 @@ impl<C: Cache> View<C> {
         self.peer.send_rpc_notification("add_scopes", &params);
     }
 
+    pub fn edit(&self, edit: PluginEdit) {
```

Comment:
instead of taking a `PluginEdit`, this should take just the arguments determined by the plugin (such as the delta, priority, etc) and then construct the `PluginEdit` in place.